ocs_ci/ocs/registry.py

```diff
@@ -279,3 +272,23 @@ def image_rm(registry_path):
     ocp_obj = ocp.OCP()
     ocp_obj.exec_oc_debug_cmd(node=master_list[0], cmd_list=cmd_list)
     logger.info(f"Image {registry_path} rm successful")
+
+
+def check_image_in_registry(image_url):
+    """
+    Function to check either image present in registry or not
+
+    Args:
+        image_url (str): Image url to be verified
+
+    Returns:
+        True : Returns True if present
+
+    """
+    output = image_list_all()
+    output = output.split("\n")
+    if any(image_url in i for i in output):
+        logger.info("Image URL present")
+        return True
```

Comment:
Instead of returning true can we assert here if image_url present ?

Reply:
Not asserting in-case of image present, but handling the failure in exception